.github/workflows/release.yml

```
@@ -55,16 +58,22 @@ jobs:
 
   release-sdk:
     needs: prepare-release
+    permissions:
+      contents: write
     uses: ./.github/workflows/release-project-in-dir.yml
     secrets: inherit
     with:
-      version_branch: ${{needs.prepare-release.outputs.version_branch}}
-      project_dir: '.'
+      version_branch: ${{ needs.prepare-release.outputs.version_branch }}
+      release_tag: ${{ needs.prepare-release.outputs.release_tag }}
```

actionlint (1.7.12)

[error] input "release_tag" is not defined in "./.github/workflows/release-project-in-dir.yml" reusable workflow. defined inputs are "project_dir", "version_branch"

(workflow-call)